web/src/user_groups.ts

```diff
@@ -358,6 +358,32 @@ export function get_recursive_group_members(target_user_group: UserGroup): Set<n
     return members;
 }
 
+export function is_group_larger_than(target_user_group: UserGroup, max_size: number): boolean {
+    const members = new Set(target_user_group.members);
+    if (members.size > max_size) {
+        return true;
+    }
```

Comment:
This still copies the entire set, so does linear work in the size of the group itself. This could be avoided by first checking the size. I'll tweak.